templates/base.html.twig

```diff
@@ -10,88 +10,70 @@
                               <path stroke-linecap="round" d="M16.5 12a4.5 4.5 0 11-9 0 4.5 4.5 0 019 0zm0 0c0 1.657 1.007 3 2.25 3S21 13.657 21 12a9 9 0 10-2.636 6.364M16.5 12V8.25" />
                             </svg>
-                            <span class="ms-2">Domains</span>
+                            <span class="ms-1">Domains</span>
                           </a>
-                        </li>
                         {% if is_granted('ROLE_ADMIN') %}
-                        <li class="nav-item">
-                          <a class="nav-link{% if menuactive == "users" %} active{% endif %}" href="{{ path('app_users') }}">
+                          <a title="Users" class="text-nowrap nav-link{% if menuactive == "users" %} active{% endif %}" href="{{ path('app_users') }}">
                             <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
                               <path stroke-linecap="round" stroke-linejoin="round" d="M15 19.128a9.38 9.38 0 002.625.372 9.337 9.337 0 004.121-.952 4.125 4.125 0 00-7.533-2.493M15 19.128v-.003c0-1.113-.285-2.16-.786-3.07M15 19.128v.106A12.318 12.318 0 018.624 21c-2.331 0-4.512-.645-6.374-1.766l-.001-.109a6.375 6.375 0 0111.964-3.07M12 6.375a3.375 3.375 0 11-6.75 0 3.375 3.375 0 016.75 0zm8.25 2.25a2.625 2.625 0 11-5.25 0 2.625 2.625 0 015.25 0z" />
                             </svg>
-                            <span class="ms-2">Users</span>
+                            <span class="ms-1">Users</span>
                           </a>
-                        </li>
-                        <li class="nav-item">
-                          <a class="nav-link{% if menuactive == "logs" %} active{% endif %}" href="{{ path('app_logs') }}">
+                          <a title="Logs" class="text-nowrap nav-link{% if menuactive == "logs" %} active{% endif %}" href="{{ path('app_logs') }}">
                             <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
                               <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 14.25v-2.625a3.375 3.375 0 00-3.375-3.375h-1.5A1.125 1.125 0 0113.5 7.125v-1.5a3.375 3.375 0 00-3.375-3.375H8.25m0 12.75h7.5m-7.5 3H12M10.5 2.25H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 00-9-9z" />
                             </svg>
-                            <span class="ms-2">Logs</span>
+                            <span class="ms-1">Logs</span>
+                          </a>
+
+                          <a title="Check" class="text-nowrap nav-link nav-link-extra link-secondary" href="{{ path('app_checkmailnow') }}">
+                            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
```

Comment:
personally i'd prefeer the outlined - inbox-arrow-down icon from the heroicons set

Reply:
Fine by me.